upgrade-notes/storefront_20241206_134303.md

```diff
@@ -0,0 +1,5 @@
+#### add watchdog ([#3640](https://github.com/shopsys/shopsys/pull/3640))
+
+- customer can create watchdog for product from product detail page
```

Comment:
can you please add backend upgrade notes? at least there are some changes in the project-base